packages/wallet-ui/src/services/useStarkNetSnap.ts

```diff
@@ -380,7 +380,7 @@ export const useStarkNetSnap = () => {
       }
 
       //Set the deploy transaction
-      const deployTransaction = storedTxns.find((txn: Transaction) => txn.txnType === 'deploy');
+      const deployTransaction = storedTxns.find((txn: Transaction) => txn.txnType.toLowerCase() === 'deploy');
```

Comment:
We could change the `=== 'deploy' ` by `=== VoyagerTransactionType.DEPLOY`

Reply:
@Julink-eth done, thx!